src/OpenLoco/Ui/ViewportInteraction.cpp

```diff
@@ -407,11 +407,35 @@ namespace OpenLoco::Ui::ViewportInteraction
     // 0x004CDB2B
     InteractionArg rightOver(int16_t x, int16_t y)
     {
+        if (OpenLoco::isTitleMode())
+            return InteractionArg{};
+
+        // Default flags
+        auto filterInteractionsFlags = 0 | InteractionItemFlags::surface | InteractionItemFlags::water;
```

Comment:
This is an exclusive filter, rather than an inclusive filter, right? Perhaps we could clarify that, e.g.:
```suggestion
        // Interaction types to exclude by default.
        auto interactionsToExclude = InteractionItemFlags::surface | InteractionItemFlags::water;
```
Naming things is hard; @duncanspumpkin may have a better suggestion.

Reply:
Yep, makes more sense like this.